resources/views/dictionary/lookup.blade.php

```diff
@@ -26,6 +26,11 @@
                 <h3><b><a href="{{ route('word.lookup',[$word->word]) }}">{{ strtolower($word->word) }}</a></b></h3>
                 <p class="definition">{{ $word->translation }}</p>
                 <p class="etymology">{{ $word->etymology }}</p>
+                @if(stripos($word->filter, 'noncanon') !== false)
+                    @if($dictionary == ucfirst($dictionary))
```

Comment:
A creative way to do it I must say 😄 but simply doing `@if(!isset($dictionary))` here works as well, and is probably better for readability. Unless there is a specific reason for doing it this way?

```suggestion
                    @if(!isset($dictionary))
```

Reply:
None. That does make things more readable. I've been up far too long and should really sleep is probably why. I can push that for readability before I actually get some sleep. Cause readability is best practice.